src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx

```diff
@@ -77,8 +77,12 @@ function TitleSection({
 }) {
   return (
     <View testID="TitleSection" onLayout={onLayout} style={styles.titleContainer}>
-      <TokenIcons tokensInfo={tokensInfo} />
-      <Text style={styles.title}>{title}</Text>
+      <View style={styles.titleTokenContainer}>
+        <View style={styles.titleTokenIconContainer}>
```

Comment:
for my understanding, what's the purpose of this extra wrapper View?

Reply:
Wrapped the token icon in an extra view to shrink it down; otherwise, it'll take up the whole line.